### DISCUSSION TITLE
ValueError: fp16 mixed precision requires a GPU -- running latest on a Mac

### DISCUSSION THREAD

**Discussion by bobpuffer1** [bmaltais/kohya_ss#1036] [Q&A]
MBP M2 Max with 64gb shared RAM
Pulled today

**Comment by bobpuffer1** [comment]
I deleted and re-pulled. Everything starts up OK but as soon as I click to create a LoRA, I get the same error having to do with GPU.

**Reply by vkbest** [reply]
No current support in the script for Fp16 or mixed precision in Mac. You have to select FP32
